device_frame/example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
```

Comment:
Your PR is about the iPhone 12 ratio. Please exclude unrelated changes such as this one.